### PR TITLE
an/SKY-67: Design 404.html Page and Set it Up

### PR DESCRIPTION
**Pull request by anurmamedov**
*Design of 404.html page, update url and view

image

### CODE REVIEW

File: templates/profile-details.html
Comment (anurmamedov) [inline templates/profile-details.html:1]: when you create new profile, on position for image you can see "min" writing , looks like an error.  That's why changed to "#"

File: profiles/views.py
Comment (anurmamedov) [inline profiles/views.py:1]: removed http404 , had a problem to redirect on 404.html page. Finally fixed

File: posts/views.py
Comment (anurmamedov) [inline posts/views.py:1]: removed below , no functionality

File: main/views.py
Comment (anurmamedov) [inline main/views.py:1]: removed none of use lines

**Comment by anurmamedov** [comment]
button upgraded to bootstrap type 

image

**Review by Aymammet — APPROVED** [review]

**Comment by anurmamedov** [comment]
conversations resolved

**Pull request merged**